src/content/docs/agile-handbook/heroing/what-is-a-hero.mdx

```diff
@@ -1,14 +1,19 @@
 ---
-title: "What is a hero?"
+title: "What is a hero? What is a sidekick?"
 template: basicDoc
 topics:
   - Docs agile handbook
-freshnessValidatedDate: never
+freshnessValidatedDate: 2024-02-06
 ---
 
 "Hero" is a common term across New Relic for a dedicated, interruptible person who acts as an interface for a team. When you're a docs hero, you're the face of the team.
 
-We have two heroes at any given time: the GitHub hero (for issues and pull requests) and the Slack hero (for questions through Slack). We also have a second-shift hero to support EMEA Relics. Every hero's job is to keep things moving for Relics and users. We change GitHub and  heroes once a day (we used to do weekly shifts, but we've found daily shifts reduce hero burnout).
+We have two rotating roles at any given time: 
+
+* The Hero handles Slack questions, reviews pull requests, and triages GitHub issues
+* The CSAT Sidekick handles customer feedback coming in through Jira. 
+
+Both roles are here to keep things moving for Relics and users. and to buffer the rest of the team from interrupts. We change hero and a sidekick once a week (we used to do daily shifts, but we've found the daily cadence to be too disruptive and lead to work that bleeds over to the next day).
```

Comment:
```suggestion
Both roles are here to keep docs requests moving for Relics and users, and to give the team a buffer from interruptions. We change hero and a sidekick once a week (we used to do daily shifts, but we've found the daily cadence to be too disruptive and lead to work that bleeds over to the next day).
```

Reply:
I might consider editing the last parens to just describe how we do it currently. Something like "We currently use weekly hero shifts to mitigate work that bleeds into the next day, leading to smaller tasks to fall through the cracks."